ReactWindows/ReactNative.Shared/UIManager/UIViewOperationQueue.cs

```diff
@@ -480,19 +481,16 @@ private void OnRendering(object sender, object e)
                 }
             }
 
+            Action[] actions = EMPTY_ACTIONS_ARRAY;
             lock (_gate)
             {
-                try
-                {
-                    foreach (var batch in _batches)
-                    {
-                        batch();
-                    }
-                }
-                finally
-                {
-                    _batches.Clear();
-                }
+                actions = _batches.ToArray();
```

Comment:
yep, I will try it on android as well 👍 
Weird that I didn't noticed it there but noticed in the windows fork :D